picard/ui/itemviews/basetreeview.py

```diff
@@ -91,6 +91,7 @@
 from picard.ui.collectionmenu import CollectionMenu
 from picard.ui.enums import MainAction
 from picard.ui.filter import Filter
+from picard.ui.itemviews.match_quality_column import MatchQualityColumn, MatchQualityColumnDelegate
```

Comment:
Is this the new style for `import` formatting because of ruff?  Wasn't our standard to format multiple imports as a tuple with each import on a separate line?

Reply:
Yes, but there was an isort rule `force_grid_wrap = 2` (along with other formatting rules) in the `pyproject.toml` file that were different from the current ruff configuration.  Thus the reason for my question, because the include style is now different.

Reply:
> Ruff prefers `black` style (can't change it).

I see that, and it appears that the ruff development team have no intention of supporting isort formatting.  That's unfortunate because the way that we used to format the imports made a lot more sense (in my opinion) when reviewing git diffs.

> Anyway that's a `pyproject.toml`/formatting change, unrelated to this PR.

Agreed.  It's just that this PR happens to be where I noticed the changes.

Reply:
I generally agree with @zas. I'm also not really happy with the black style. The changing of import style based on line length is rather annoying and can lead to unnecessary and hard to read changed lines in diffs rather randomly. We had quite lengthy discussions about this when we introduced isort managing the imports, and the current system of having multiple imports on separate lines worked really well.

But if we really can't change it I would prefer the convenience of `ruff format` over my personal taste.

We should do a complete formatting run in a separate PR and check the result. Maybe there is more changes that we need to discuss and probably tweak one way or another.

Reply:
@phw has summarized my thoughts exactly.

Reply:
I ran `ruff format` on current master branch, and created a PR for us to review changes: https://github.com/metabrainz/picard/pull/2697